svg/elements/circle.json

```diff
@@ -0,0 +1,195 @@
+{
+  "svg": {
+    "elements": {
+      "circle": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/SVG/Element/circle",
+          "support": {
+            "webview_android": {
+              "version_added": "3"
+            },
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "1.5"
+            },
+            "firefox_android": {
+              "version_added": "4"
+            },
+            "ie": {
+              "version_added": "9"
+            },
+            "opera": {
+              "version_added": "8"
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": "3.1"
+            },
+            "safari_ios": {
+              "version_added": "3.1"
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "cx": {
```

Comment:
This feature has an mdn_url now, will it stay after your svg docs reorg? https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/cx
If yes, this should have that url as an mdn_url, if not, this is fine.

Reply:
No, I plan to remove/redirect all attribute pages that are specific to a given element. That means that all attribute in an element compat data object won't have a page after the reorg of the doc.